salt/haproxy/init.sls

```diff
@@ -0,0 +1,105 @@
+{% set pnda_domain = pillar['consul']['data_center'] + '.' + pillar['consul']['domain'] %}
+{% set grafana_server_node = salt['pnda.get_hosts_for_role']('grafana')[0] + '.' +pillar['consul']['node'] + '.' + pnda_domain %}
+{% set jupyter_server_node = salt['pnda.get_hosts_for_role']('jupyter')[0] + '.' +pillar['consul']['node'] + '.' + pnda_domain %}
+{% set pnda_home_dir = pillar['pnda']['homedir'] %}
+{% set haproxy_config_dir = '/etc/haproxy' %}
```

Comment:
Most other pnda components have their config file in the component directory. Can we keep consistence here? I believe you can point haproxy to any configuration file location.